Tests/FoundationTests/FileManagerExtensionsTests.swift

```diff
@@ -80,6 +80,28 @@ final class FileManagerExtensionsTests: XCTestCase {
         } catch {}
     }
 
+    func testEncode() {
+        let point = CGPoint(x: 1, y: 2) //CGPoint conforms to Encodable
+        do {
+            let temporaryDirectoryURL = FileManager.default.temporaryDirectory
+            let temporaryFilename = ProcessInfo().globallyUniqueString
+            let temporaryFileURL = temporaryDirectoryURL.appendingPathComponent(temporaryFilename)
+
+            XCTAssertNoThrow(try FileManager.default.encode(point, to: temporaryFileURL))
```

Comment:
No need for the assertion here since you're `catch`-ing. It won't make sense to run the rest of the test anyway if this fails, so better that it be caught.